src/provider/systemdmanager/systemdunit.rs

```diff
@@ -554,6 +575,40 @@ mod test {
             [Install]
             WantedBy=multi-user.target"#}
     )]
+    #[case::with_container_on_session_bus_stoptimeout(
```

Comment:
In this test case it does not matter if there are containers or which session bus is configured. So a more focused name could be chosen like `set_termination_timeout`.

The other test cases test several things together because `systemdunit` has too many layers incorporated (mapping from pod spec to service unit, creation of a systemd unit, creation of an INI file). Creating focused test cases would result in too many and too verbose test cases so test cases were merged together. It is okay to have this test case but it should be focused then. This module should really be refactored someday.

Reply:
renamed